taskboot/push.py

```diff
@@ -138,6 +141,7 @@ def heroku_release(target, args):
                 'Authorization': f"Bearer {config.heroku['password']}",
             },
     )
+    logger.info("Heroku deployment answer: %s", r.text)
```

Comment:
Either remove or set it as debug, i don't like having the full output with Application ids exposed by default.

It would be even better to validate the output (presence of ids)

Reply:
I'm afraid we cannot validate much on the heroku answer. The heroku answer contains:
- The app name and id.
- Each worker command but we don't extract that from the images yet.
- An id per worker which doesn't seems linked to the docker image id.

The rest of the fields are dates, number and dyno size.